test/test_common/network_utility.cc

```diff
@@ -69,5 +71,44 @@ Address::InstanceConstSharedPtr findOrCheckFreePort(const std::string& addr_port
   return instance;
 }
 
+Address::InstanceConstSharedPtr getSomeLoopbackAddress(Address::IpVersion version) {
+  if (version == Address::IpVersion::v4) {
+    // Pick a random address in 127.0.0.0/8.
+    Runtime::RandomGeneratorImpl rng;
```

Comment:
Not as things currently stand. See RandomGeneratorImpl::threadLocalGenerator, which has a seed based on the current time AND thread id.